config/rbac/role.yaml

```diff
@@ -85,6 +85,18 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - cluster.x-k8s.io
+  resources:
+  - machinesets
+  verbs:
+  - create
```

Comment:
do we need all of these permissions? Maybe list, get, watch would be enough ?

Reply:
IAlthough errors logs complain with listing problem only,  I have enabled all of those permissions because that's what was done for KCP, MD in previous patch, so I am not sure.